ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -113,6 +113,11 @@ export function searchOptionsToSearchFilter(searchOptions: GlobalSearchOption[])
     return searchFilter;
 }
 
+// TODO Comments and tests
+export function isCompleteSearchFilter(searchFilter: SearchFilter) {
+    return Object.values(searchFilter).every((o) => o !== '');
```

Comment:
I think the `SearchFilter` type might also be an array of strings. We can then probably check for no value, an empty string (if a string), and an empty array (if an array) as well

Reply:
Good call. I think the array case only occurs when the `qs` library parses a parameter with multiple values, always resulting in an array with length >= 2. This is probably an implementation detail though as there is nothing stopping other code paths from modifying the array. I'll update this to make it safer.